### PR TITLE
[RUSH-1363] Borders token + housekeeping

### PR DESCRIPTION
**Pull request by karuto**
### Overview
This PR introduces the borders token, which will store border related attributes, such as border radius, border width, border color, border style, etc. 

This PR just creates the token to lay the ground, we will work with designers to fill more values in there.

This PR also does some housekeeping:
 - removes the last remaining personal developer info from author field;
 - removes the homepage field in `package.json` because individual token doesn't have readme anymore;
 - adjusts the order of tokens in main readme to be consistent with the alphabetical order of folders.

### CODE REVIEW

**Comment by karuto** [comment]
@matteofigus what should I do for publishing when I create a new token? Should I run normal `npm publish` or use `lerna`(`npm run publish`)? I don't think the docs have this scenario explained, I am happy to add it in once you teach me. 😃

File: OTKit/otkit-borders/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "otkit-borders",
+  "version": "1.0.0",
```
Comment (matteofigus) [inline OTKit/otkit-borders/package.json]: The convention I found good, is to make a new package having `0.0.1` - once merged, we can publish as `1.0.0` with a `npm run publish` on the root - if you have 1.0.0 it forces you to bump it to something like 1.0.1 which is annoying

Comment (karuto) [reply OTKit/otkit-borders/package.json]: Ok that makes sense. I will put this note in readme for people want to create new tokens in future.

**Comment by karuto** [comment]
@matteofigus can you review again, I fixed the version number.

**Pull request merged**